docs/app/Components/ComponentDoc/ComponentProps.js

```diff
@@ -3,11 +3,6 @@ import React, { Component, PropTypes } from 'react'
 
 import { Label, Table } from 'src'
 
-const descriptionExtraStyle = {
-  fontSize: '0.95em',
-  color: '#777',
-}
-
```

Comment:
It looks like the enums support was mistakenly removed from this file.  Bad merge?